src/js/themes/base.js

```diff
@@ -783,12 +783,18 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       xlarge: { ...fontSizing(2) },
       xxlarge: { ...fontSizing(4) },
     },
-    // textArea: {
-    //   extend: undefined,
-    // },
-    // textInput: {
-    //   extend: undefined,
-    // },
+    textArea: {
+      disabled: {
```

Comment:
I'm thinking maybe we want the `extend` object to be available in `disabled`. hence the user can customize it as he wishes. thoughts?

Reply:
Do you think it will be tricky?